ui/raidboss/data/06-ew/raid/p9s.txt

```diff
@@ -29,20 +46,178 @@ hideall "--sync--"
 60.4 "Dualspell (Fire/Ice)" sync / 1[56]:[^:]*:Kokytos:8154:/
 63.7 "--sync--" sync / 1[56]:[^:]*:Kokytos:(8122|8123):/
 68.1 "Blizzard III" sync / 1[56]:[^:]*:Kokytos:(8157|8159):/
-68.1 "Pile Pyre" sync / 1[56]:[^:]*:Kokytos:(8156|8158):/
+68.1 "Pile Pyre" #sync / 1[56]:[^:]*:Kokytos:(8156|8158):/
```

Comment:
If there's multiple lines for an ability but they all happen at the same time, there's no need to comment out the syncs (since they'll all go to the same place).  Could you remove this comment and the other ones on thunder and pile pyres below?

Reply:
I commented out these syncs because it was causing the `test_timeline` utility to complain about missed syncs depending on which ability (Blizzard III or Pile Pyre/Thunder III) showed up in the log first.  Do you still want them uncommented?

Reply:
Yeah, I think that's fine.  We should fix test_timeline to handle multiple syncs at the same time, personally.  Feel free to file an issue for this so it doesn't get lost.